docs/intro/user_types.rst

```diff
@@ -0,0 +1,123 @@
+.. _usertypes:
+
+Tell me what you are and I tell you where to start
+--------------------------------------------------
+
+The DataLad Handbook has grown into an extensive document. Depending on your
+use case for DataLad, you may not want to read *all* of the content there is.
+This section tries to be your guide through the handbook.
+
+.. figure:: ../artwork/src/user_types.svg
+
+If you can identify with one of the user types listed below, check out the
+recommended sections.
+
+**1** Independent but intrigued
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+"I don't have a particular use case for DataLad (yet), but I want to know what
+it is all about".
+
+Start by reading sections :ref:`executive_summary` to get a high-level overview
+about DataLad's functionality, and continue with section :ref:`philo` for
+a short introduction to the fundamental principles of the software.
+Afterwards, you may want to skim through the different
+`usecases <../usecases/intro>`_ to see whether one catches your attention.
+The :ref:`cheat`, finally, can give you concrete command overviews.
+
+**2** Eager to try all of the things!
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+"I so want to try out *all* of DataLad!"
+
+Awesome, and Welcome! The complete :ref:`basics-intro` part of the book was
+written for you. If you read it from start to end, you will become a DataLad
+expert. Don't forget to `install <installation>`_ DataLad first, though.
+And if the Basics are not enough, continue right into the :ref:`usecase-intro`
+afterwards.
+
+**3** Seeking help
+^^^^^^^^^^^^^^^^^^
+
+"I ran into a problem and hoped the book could help".
+
+The section :ref:`help` may give you a good general overview on what to do if
+you encountered a problem. If you're dealing with file system operations,
+:ref:`filesystem` could be a resource to help you, and for all things configuration,
+the chapter :ref:`chapter_config` is your place to go to. If you are confused by
+symlinks or "permission denied" error in your dataset, checkout section
+:ref:`symlink` for some Basics on :term:`git-annex`.
+
+If you're seeking help with regard to large datasets, checkout section
+
+.. todo::
+
+   write section on scaling up
+
+If none of this helps, don't hesitate to file an issue or post your question
+to `Neurostars.org <https://neurostars.org>`_. We are happy to help!
```

Comment:
I think might be worth to explicitly point to "Search" on the handbook page.  Also, since "ran into a problem", it is not unlikely that user hit a bug.  Search through bug reports on github would be useful but "help" section refers only to neurostars (it seems nowhere in the book there is a reference to https://github.com/datalad/datalad/issues on purpose or not). Situation is a bit tricky since issues could be due to extensions, but at least searching through datalad organization could lead to an answer. 

Also, related, hopefully eventually https://github.com/datalad/datalad/issues/3942 would be pushed forward, so users running into the bugs would get (open or closed) issues identified automagically

Reply:
true, very good points.